src/cloud-api-adaptor/pkg/securecomms/sshproxy/sshproxy.go

```diff
@@ -61,67 +61,30 @@ type Inbounds struct {
 	list []*Inbound
 }
 
-func (outbounds *Outbounds) Add(tag string) {
-	splits := strings.Split(tag, ":")
-	var port, host, name, phase string
-	if len(splits) == 3 {
-		phase = splits[0]
-		name = splits[1]
-		port = splits[2]
-		host = "127.0.0.1"
-	} else if len(splits) == 4 {
-		phase = splits[0]
-		name = splits[1]
-		host = splits[2]
-		port = splits[3]
-	} else {
-		log.Fatalf("Outbound Add wrong Tag: %s", tag)
-	}
-	if _, err := strconv.ParseUint(port, 10, 16); err != nil {
-		log.Fatalf("Outbound Add Port '%s' - Err: %v", port, err)
-	}
-	if phase != "A" && phase != "K" && phase != "B" {
-		log.Fatalf("Outbound Add ilegal Phase '%s'", phase)
-	}
+func (outbounds *Outbounds) Add(port int, host, name, phase string) {
 	outbound := &Outbound{
 		Phase:   phase,
 		Name:    name,
-		OutAddr: host + ":" + port,
+		OutAddr: fmt.Sprintf("%s:%d", host, port),
 	}
 	outbounds.list = append(outbounds.list, outbound)
 }
 
-func (inbounds *Inbounds) Add(tag string, wg *sync.WaitGroup) (string, string, error) {
-	splits := strings.Split(tag, ":")
-	if len(splits) != 3 {
-		return "", "", fmt.Errorf("Inbound Add wrong Tag: %s", tag)
-	}
-	phase := splits[0]
-	name := splits[1]
-	inPort := splits[2]
-
-	if phase != "A" && phase != "K" && phase != "B" {
-		log.Fatalf("Inbound Add ilegal Phase '%s'", phase)
-	}
-	port, err := strconv.ParseUint(inPort, 10, 16)
-	if err != nil {
-		return "", "", fmt.Errorf("Inbound Add Port '%s' - Err: %v", inPort, err)
-	}
-
+func (inbounds *Inbounds) Add(inPort int, name, phase string, wg *sync.WaitGroup) (string, error) {
 	tcpAddr := &net.TCPAddr{
 		IP:   net.IPv4(127, 0, 0, 1),
-		Port: int(port),
+		Port: inPort,
 	}
 
 	tcpListener, err := net.ListenTCP("tcp", tcpAddr)
 	if err != nil {
-		return "", "", fmt.Errorf("Inbound Failed to Listen to Host: %s Port '%s' - Err: %v", name, inPort, err)
+		return "", fmt.Errorf("inbound Failed to Listen to Host: %s Port '%d' - Err: %v", name, inPort, err)
```

Comment:
I find the capitalization very strange here (both old and new).

Why use Americal Journal Title Capitalization instead of reGulaR noRmal capItalization? :-)

If the first word is supposed to be the class name, then it's capitalized and with an s.

Reply:
```suggestion
		return "", fmt.Errorf("Inbounds Add: failed to listen to lost %s:%d, error %v", name, inPort, err)
```

Reply:
I will scan and fix capitalizations. 
Golang does not like errors to start with a capital letter though since errors tend to be concatenated.